script.js

```diff
@@ -1,70 +1,96 @@
-let Books = [];
-function NewbookRemove(RButton) {
-  const BookContainer = document.getElementById(RButton.id);
-  BookContainer.parentNode.removeChild(BookContainer);
-  /* eslint-disable */ 
-  const BooksNew = Books.filter(function (item) {
-    return item.id != RButton.id;
-  });
-  /* eslint-enable */
-  Books = BooksNew;
-  localStorage.clear();
-  const BookList = JSON.stringify(BooksNew);
-  // eslint-disable-next-line no-unused-vars
-  localStorage.setItem('library', BookList);
-}
-// eslint-disable-next-line no-unused-vars
-function Clear() {
-  localStorage.clear();
-  Books = [];
-  document.getElementById('list_container').innerHTML = '';
-}
-function CreateHtml(IdBook) {
-  const newdiv = document.createElement('div');
-  newdiv.setAttribute('id', IdBook);
-  const title = document.createElement('p');
-  const author = document.createElement('p');
-  const RemoveButt = document.createElement('button');
-  RemoveButt.addEventListener('click', () => { NewbookRemove(RemoveButt); });
-  RemoveButt.textContent = 'remove';
-  RemoveButt.setAttribute('id', IdBook);
-  return [newdiv, title, author, RemoveButt];
-}
-function SaveLocally(BookArray) {
-  const BookList = JSON.stringify(BookArray);
-  localStorage.setItem('library', BookList);
-}
-function RetrieveOld() {
-  Books = JSON.parse(localStorage.getItem('library')) || [];
-  SaveLocally(Books);
-}
+class BookList {
+  constructor(Books) {
+    this.BookListCollection = Books;
+  }
+
+  clear() {
+    localStorage.clear('library');
+    document.getElementById('list_container').innerHTML = '';
+    this.BookListCollection = [];
+  }
 
-function ShowBook() {
-  RetrieveOld();
-  const BooksCurrent = JSON.parse(localStorage.getItem('library'));
   /* eslint-disable */
-  BooksCurrent.forEach(element => {
-    const x = CreateHtml(element.id);
-    x[1].innerHTML = `Title:   ${element.title}`
-    x[2].innerHTML = `Author:   ${element.author}`;
-    document.getElementById('list_container').appendChild(x[0]).appendChild(x[1]);
-    document.getElementById('list_container').appendChild(x[0]).appendChild(x[2]);
-    document.getElementById('list_container').appendChild(x[0]).appendChild(x[3]);
-  });
+  AddBook(Book) {
+    this.BookListCollection.push(Book);
+    const TitleP = document.createElement('p');
+    const AuthorP = document.createElement('p');
+    const RButton = document.createElement('button');
+    RButton.addEventListener('click', () => { this.RemoveBook(Book.id); });
```

Comment:
- [ ] Kindly follow the camelCase naming convention and rename function and variable names to reflect this. So AddBook should be `addBook` and `this.BookListCollection` should be `this.bookListCollection`.